AppFog/deploy-python-application.md

```diff
@@ -27,7 +27,7 @@ To develop using Python you must have the its [runtime interpreter](https://wiki
 You will also need [Pip](https://pip.pypa.io/en/stable/installing/) installed in your development environment.
 
 #### Downloading Simple Example Application
-The sample app used in this article is [attached](../attachments/af-python-django-sample.zip).
+The sample app used in this article is [attached](../../attachments/af-python-django-sample.zip).
```

Comment:
Change to `The sample app used in this article is attached below.`

You cannot reference attachments in the body of the article in this way.